pkg/auth/authTokensGet.go

```diff
@@ -37,6 +38,48 @@ func GetTokens(
 	return err
 }
 
+func GetTokensByLoginId(
+	apiClient *galasaapi.APIClient,
+	console spi.Console,
+	loginId string,
+) error {
+
+	authTokens, err := getAuthTokensByLoginIdFromRestApi(apiClient, loginId)
+
+	if err == nil {
+		summaryFormatter := tokensformatter.NewTokenSummaryFormatter()
+
+		var outputText string
+		outputText, err = summaryFormatter.FormatTokens(authTokens)
+
+		if err == nil {
+			console.WriteString(outputText)
+		}
+	}
+
+	return err
+}
+
+func getAuthTokensByLoginIdFromRestApi(apiClient *galasaapi.APIClient, loginId string) ([]galasaapi.AuthToken, error) {
+	var context context.Context = nil
+	var authTokens []galasaapi.AuthToken
+
+	validateLoginIdFlag(loginId)
+
+	tokens, resp, err := apiClient.AuthenticationAPIApi.GetTokens(context).LoginId(loginId).Execute()
+
+	if err != nil {
+		log.Println("getAuthTokensFromRestApi - Failed to retrieve list of tokens from API server")
+		err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_RETRIEVING_TOKEN_LIST_FROM_API_SERVER, err.Error())
+	} else {
+		defer resp.Body.Close()
+		authTokens = tokens.GetTokens()
+		log.Printf("getAuthTokensFromRestApi -  %v tokens collected", len(authTokens))
+	}
+
```

Comment:
This looks duplicated from `getAuthTokensFromRestApi` function - we can pass in the loginId parameter into that function and if that's been set, then we can add the loginId query parameter onto the request - see `runs get` for an example.